core/rawdb/accessors_metadata.go

```diff
@@ -64,8 +64,8 @@ func ReadChainConfig(db ethdb.KeyValueReader, hash common.Hash) *params.ChainCon
 		log.Error("Invalid chain config JSON", "hash", hash, "err", err)
 		return nil
 	}
-	if config.Kanvas != nil {
-		config.Clique = nil // get rid of legacy clique data in chain config (kanvas goerli issue)
+	if config.Kroma != nil {
+		config.Clique = nil // get rid of legacy clique data in chain config (kroma goerli issue)
```

Comment:
it it better to remove this comment? because we don't run the network on goeril.

Reply:
I think so.